templates/includes/fxa_sign_in_link.html

```diff
@@ -1,7 +1,8 @@
+{% load i18n %}
 {% load socialaccount %}
 <div class="sign-in-area">
-    <a class="button sign-in-btn" href="{% provider_login_url "fxa" %}?next={{ request.path }}">Sign in with <i class="fa fa-firefox"></i> Firefox Account</a>
+    <a class="button sign-in-btn" href="{% provider_login_url "fxa" %}?next={{ request.path }}">{% blocktrans %}Sign in with <i class="fa fa-firefox"></i> Firefox Account {% endblocktrans %}</a>
```

Comment:
Not sure if it matters for localizers, but we could remove the space after the word *Account*.